bots/twitch/sogebot/egg-soge-bot.json

```diff
@@ -17,13 +17,13 @@
     "config": {
         "files": "{\r\n    \".env\": {\r\n        \"parser\": \"properties\",\r\n        \"find\": {\r\n            \"PORT\": \"{{server.build.default.port}}\"\r\n        }\r\n    }\r\n}",
         "startup": "{\r\n    \"done\": \"WebPanel is available at\"\r\n}",
-        "logs": "{}",
-        "stop": "^^C"
+        "logs": "{\r\n    \"custom\": false,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
+        "stop": "^C"
```

Comment:
^^C is a current workaround regarding catching termination signals when using eval for startup parsing. ^C doesn't get passed, and the server will be stuck stopping until Pterodactyl kills it